doc/classes/FontConfig.xml

```diff
@@ -0,0 +1,279 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FontConfig" inherits="Resource" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		FontConfig class font configuration used to draw text.
+	</brief_description>
+	<description>
+		[codeblocks]
+		[gdscript]
+		var fc = FontConfig.new()
+		fc.set_font(load("res://BarlowCondensed-Bold.ttf"))
+		fc.set_size(64)
```

Comment:
Is this correct in this version with separate font size?

Reply:
No, fixed.